tests/TestRunner.Core/TestInstrumentation.cs

```diff
@@ -292,22 +296,21 @@ protected virtual IList<TestAssemblyInfo> GetTestAssemblies ()
 
 			if (TestAssemblyDirectories != null && TestAssemblyDirectories.Count > 0) {
 				foreach (string adir in TestAssemblyDirectories)
-					GetTestAssembliesFromDirectory (adir, TestAssembliesGlobPattern, ret);
+					GetTestAssembliesFromDirectory (adir, ret);
 			}
 
 			return ret;
 		}
 
-		protected virtual void GetTestAssembliesFromDirectory (string directoryPath, string globPattern, IList<TestAssemblyInfo> assemblies)
+		protected virtual void GetTestAssembliesFromDirectory (string directoryPath, IList<TestAssemblyInfo> assemblies)
```

Comment:
Please keep the current logic and implement the enumerator method as suggested above.